### PR TITLE
feat: add support for Kubernetes 1.15.12

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
See https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.15.md#changelog-since-v11511

**Issue Fixed**:

**Requirements**:

- [x] Kubernetes artifacts built and pushed by Azure Pipelines
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
/hold

This has a temporary testing commit that should be discarded before merging.

**Comment by mboersma** [comment]
This passed e2e tests for 1.15 [here](https://dev.azure.com/aks-engine/AKS-Engine/_build/results?buildId=4939&view=logs&jobId=a48fcffa-d400-5086-fd55-6a393fb8fdb7), so I'll discard the testing commit and this should be ready for review.

**Comment by jadarsie** [comment]
FYI, I may have the `-azs` image ready EOD

File: vhd/packer/install-dependencies.sh [hunk 1]
```diff
@@ -361,10 +361,9 @@ K8S_VERSIONS="
 1.16.9-azs
 1.16.8
 1.16.8-azs
+1.15.12
```
Comment (jadarsie) [inline vhd/packer/install-dependencies.sh:364]: could you please add `1.15.12-azs`?

File: vhd/packer/configure-windows-vhd.ps1 [hunk 2]
```diff
@@ -85,13 +85,11 @@ function Get-FilesToCacheOnVHD
         "c:\akse-cache\win-k8s\" = @(
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.7-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.7-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.9-1int.zip",
```
Comment (mboersma) [inline vhd/packer/configure-windows-vhd.ps1:89]: Should we add `https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-azs/windowszip/v1.15.12-azs-1int.zip` and friends to the Windows VHD? That seems to be the current URL where this archive is published.

Comment (jadarsie) [reply vhd/packer/configure-windows-vhd.ps1:89]: Yes, I have a PR ready, just waiting for v1.15.10

Comment (jadarsie) [reply vhd/packer/configure-windows-vhd.ps1:89]: Will free to update if you like

Comment (mboersma) [reply vhd/packer/configure-windows-vhd.ps1:89]: Let's do those changes in a separate PR just to keep it simple.

**Pull request merged**